docs/observability/aws/integrations/aws-ec2-cloudwatch-metrics.md

```diff
@@ -0,0 +1,206 @@
+---
+id: aws-ec2-cloudwatch-metrics
+title: AWS EC2 CloudWatch Metrics
+sidebar_label: AWS EC2 CloudWatch Metrics
+description: The Sumo Logic App for AWS EC2 (CloudWatch Metrics) allows you to display and gain insight about your EC2 instance metrics (CloudWatch) for cpu, disk, network, EBS and Health Status Check.
+---
+
+[Amazon Elastic Compute Cloud (Amazon EC2)](https://aws.amazon.com/ec2/) provides scalable computing capacity in the Amazon Web Services (AWS) Cloud. You can use Amazon EC2 to launch as many or as few virtual servers as you need, configure security and networking, and manage storage.
+
+The Sumo Logic App for AWS EC2 (CloudWatch Metrics) allows you to collect your EC2 instance metrics and display them using predefined dashboards. The App provides dashboards to display analysis of EC2 instance metrics for CPU, disk, network, EBS, and Health Status Check. Furthermore, it provides detailed insights into all CloudTrail audit events associated with EC2 instances and specifically helps identify changes, errors, and user activities.
+
```

Comment:
I'd add a note here and [here](https://help.sumologic.com/docs/observability/aws/integrations/aws-ec2-metrics/) differentiating EC2 from EC2 Cloudwatch.

Reply:
The AWS EC2 metrics page will be removed, it is no longer required. I have just added a new file "aws-ec2-cloudwatch-metrics.md". Will remove the aws-ec2-metrics.md. 
Adding more information: Note to customers is not required in this doc as they are aware EC2 Metrics will now be called CW Metrics. Earlier, In aws-ec2-metrics. md, host metrics content was copy pasted, but that's not correct. CW and Host metrics are the source for EC2 metrics, so we just need two pages: aws-ec2-cloudwatch-metrics.md and aws-host-metrics.md.